reservations/models.py

```diff
@@ -150,6 +151,12 @@ def handling_required(self):
             recurring_reservation=None,
         )
 
+    def current(self):
```

Comment:
can't figure out the meaning of this filter by its name => the effect of this is quite like the `going_to_occur`

Reply:
Good point, this filter is actually the same as `going_to_occur`. I'll remove the duplication.